packages/binding-mqtt/src/util.ts

```diff
@@ -16,7 +16,7 @@ import { createLoggers } from "@node-wot/core";
 import { MqttQoS } from "./mqtt";
 import { IClientPublishOptions } from "mqtt";
 
-const { debug, warn } = createLoggers("binding-mqtt", "mqtt-util");
+const { warn } = createLoggers("binding-mqtt", "mqtt-util");
```

Comment:
Mhh interesting. I would have said that the code checks we have in place mention unused imports...

Reply:
The warning popped up in other PRs as an additional warning not related to code being changed, which is how I noticed it – but yeah, strange that it slipped through 🤔